cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,105 @@
+
+You can submit feedback or log any bugs you find through [this survey](https://cockroachlabs.typeform.com/to/gvCcF14q).
+
+## Security
+
+### Is my cluster secure?
+
+We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
+
+### Is encryption-at-rest enabled on CockroachCloud Free?
+
+Yes. All data on CockroachCloud is encrypted-at-rest using the tools provided by the cloud provider that your cluster is running in.
+
+- Data stored in clusters running in GCP are encrypted-at-rest using [persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption).
+- Data stored in clusters running in AWS are encrypted-at-rest using [EBS encryption-at-rest](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+
+Because we are relying on the cloud provider's encryption implementation (as noted above), we do not enable CockroachDB's [internal implementation of encryption-at-rest](../v20.2/encryption.html#encryption-at-rest-enterprise). This means that encryption will appear to be disabled in the [DB Console](../v20.2/ui-overview.html), since it is unaware of cloud provider encryption.
+
+### Is my cluster isolated? Does it share resources with any other clusters?
+
+CockroachCloud Free is a multi-tenant offering and resources are shared between clusters.
+
+## Cluster maintenance
+
+### How do I add nodes?
+
+We have eliminated the concept of nodes for CockroachCloud Free. However, there is an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster. If you exceed this limit, you can create a cluster using the paid version of [CockroachCloud](create-your-cluster.html).
+
+### Can I upgrade my cluster from CockroachCloud Free to the paid version of CockroachCloud?
+
+At this time, a CockroachCloud Free cluster cannot be upgraded. In the future, you will have the ability to move from CockroachCloud Free to a paid version of CockroachCloud.
+
+### Who is responsible for backups? Can I backup my cluster?
+
+Currently, backups are not supported on CockroachCloud Free. The ability to enable backups will be supported in the future.
+
+## Product features
+
+### Are enterprise features like partitioning or change data capture available to me?
+
+At this time, change data capture and partitioning are not available on CockroachCloud Free clusters, but will be in the future.
+
+### Do you have a UI? How can I see details?
+
+All customers of our CockroachCloud service can view and manage their clusters in the [Console](https://cockroachlabs.cloud/).
+
+### Can I backup my CockroachCloud Free cluster? Does Cockroach Labs take backups of my cluster?
+
+Cockroach Labs takes full cluster backups of all CockroachCloud Free clusters for our own purposes. Currently, these backups are not available to you and you cannot backup and restore a CC Free cluster yourself. We expect to support user initiated backup and restore of free clusters in the future.
```

Comment:
This is really weird.  Above you said no backups?  Now you are saying you are keeping backups of my data "for your own purposes"?  As a user this feels very off.

(BTW my comments may all sound very adversarial, I apologize.  I am pretending I don't work here and reading this FAQ as if I were looking at someone else's FAQ.  In general I read such things with a very skeptical eye, esp. when something is "free", since nothing in this world is free)